metrics/metrics.go

```diff
@@ -52,18 +54,15 @@ func GetCommand(args []string) string {
 	onlyFlags := false
 	for _, arg := range args {
 		if arg == "--help" {
-			return ""
+			result = strings.TrimSpace(arg + " " + result)
+			continue
 		}
 		if arg == "--" {
 			break
 		}
 		if isCommandFlag(arg) || (!onlyFlags && isCommand(arg)) {
-			if result == "" {
-				result = arg
-			} else {
-				result = result + " " + arg
-			}
-			if !isManagementCommand(arg) {
+			result = strings.TrimSpace(result + " " + arg)
```

Comment:
I don't know this code well, why did you remove `if result == ""`?

Reply:
Trim spaces rather than have an if to add space only between words, saves 4 lines of code

Reply:
I missed the addition of the TrimSpace :)

Reply:
This is happening in the background, to record metrics, and is followed by an http call to local Desktop, so I don't think we care. (and the args we manipulate are already trimmed)